From 8f5704eee10cf03caed35a4696a8e34187c972de Mon Sep 17 00:00:00 2001
From: Sivashankari Madhavan <quic_sivamadh@quicinc.com>
Date: Mon, 14 Nov 2022 15:19:13 +0530
Subject: [PATCH] ath12k: Adds the firmware SSR recovery debugs

Adds the debug print to ensure this is a simulated firmware crash
in the firmware SSR recovery case.

root@OpenWrt:/# echo 1 > /sys/kernel/debug/ath12k/<hw>/set_fw_recovery
root@OpenWrt:/# echo assert > /sys/kernel/debug/ath12k/<hw>/simulate_fw_crash
....
[  235.647604] ath12k_pci 0003:01:00.0: group_id 12 4294902926ms before
[  235.717724] ath12k_pci 0003:01:00.0: WLAN target is restarting
....

Signed-off-by: Sivashankari Madhavan <quic_sivamadh@quicinc.com>
---
 drivers/net/wireless/ath/ath12k/coredump.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/wireless/ath/ath12k/coredump.c b/drivers/net/wireless/ath/ath12k/coredump.c
index 73e7851..10a15ff 100644
--- a/drivers/net/wireless/ath/ath12k/coredump.c
+++ b/drivers/net/wireless/ath/ath12k/coredump.c
@@ -272,6 +272,7 @@ void ath12k_coredump_download_rddm(struct ath12k_base *ab)
 
 		BUG_ON(1);
 	} else {
+		ath12k_info(ab, "WLAN target is restarting");
 		ath12k_coredump_build_inline(ab, segment, num_seg);
 	}
 
-- 
2.17.1

